packages/printer-legacy/src/directive.js

```diff
@@ -0,0 +1,85 @@
+const {
+  graphql: { getDirective },
+  object: { hasProperty },
+} = require("@graphql-markdown/utils");
+const {
+  HEADER_SECTION_LEVEL,
+  HEADER_SECTION_SUB_LEVEL,
+  MARKDOWN_EOP,
+  MARKDOWN_EOL,
+} = require("./const/strings");
+const { printLink } = require("./link");
+
+function getConstDirectiveMap(type, options) {
+  if (
+    !hasProperty(options, "customDirectives") ||
+    typeof options.customDirectives !== "object" ||
+    !Object.keys(options.customDirectives).length
+  ) {
+    return undefined;
+  }
+
+  const constDirectives = getDirective(
```

Comment:
Why the name `constDirectives` instead of `customDirectives`? :thinking:

Reply:
There are 2 arguments useful for the descriptor function: one is `directiveType` of type `GraphQLDirective` which is the defined schema directive type, and the other one is `constDirectiveType` of type `ConstDirectiveNode` which is the declared directive node. Not sure why it's named `ConstDirectiveNode` in library `graphql` type system though.

I use the name `constDirectives` here to represent type `ConstDirectiveNode` and to differentiate it with the directive type definition. Maybe we could use a better naming for this?

Reply:
After some thoughts, since this is in the printer package, might naming them `custom directives` seems more appropriate? The naming for `const directive` is intended to differentiate between the declared directive node and the defined directive type, could use the naming in the utils package.

To help answer related comments, I updated the PR to use the name of `customDirective` in the printer package.

Reply:
Okay, it makes some sense. So, `constDirectives` naming would be okay in the `utils` package, and only use `customDirectives` in the `printer` package as you suggested. :+1: